javascript.js

```diff
@@ -11,4 +11,288 @@ document.getElementById('open-menu').onclick = openNav;
+ 
+<section class="Project-3" id="project-3"> 
+ <h3 class="title-p">${projectsArray[2].title}</h3>
+<p class="p-Project">${projectsArray[2].paragraph}
+</p>
+<ul class="ul-p">
+  <li class="li-p">${projectsArray[2].tecnologies[0]}</li>
+  <li class="li-p">${projectsArray[2].tecnologies[1]}</li>
+  <li class="li-p">${projectsArray[2].tecnologies[2]}</li>
+</ul>
+<button type="button" id="btn-project-3" class="btn-p">See Project</button>
+</section>
+
+
+<section class="Project-4" id="project-4"> 
+ <h3 class="title-p">${projectsArray[3].title}</h3>
+<p class="p-Project">${projectsArray[3].paragraph}
+</p>
+<ul class="ul-p">
+  <li class="li-p">${projectsArray[3].tecnologies[0]}</li>
+  <li class="li-p">${projectsArray[3].tecnologies[1]}</li>
+  <li class="li-p">${projectsArray[3].tecnologies[2]}</li>
+</ul>
+<button type="button" id="btn-project-4" class="btn-p">See Project</button>
+</section>
+
+
+<section class="Project-5" id="project-5"> 
+ <h3 class="title-p">${projectsArray[4].title}</h3>
+<p class="p-Project">${projectsArray[4].paragraph}
+</p>
+<ul class="ul-p">
+  <li class="li-p">${projectsArray[4].tecnologies[0]}</li>
+  <li class="li-p">${projectsArray[4].tecnologies[1]}</li>
+  <li class="li-p">${projectsArray[4].tecnologies[2]}</li>
+</ul>
+<button type="button" id="btn-project-5" class="btn-p">See Project</button>
+</section>
+
+<section class="Project-6" id="project-6"> 
+ <h3 class="title-p">${projectsArray[5].title}</h3>
+<p class="p-Project">${projectsArray[5].paragraph}
+</p>
+<ul class="ul-p">
+  <li class="li-p">${projectsArray[5].tecnologies[0]}</li>
+  <li class="li-p">${projectsArray[5].tecnologies[1]}</li>
+  <li class="li-p">${projectsArray[5].tecnologies[2]}</li>
+</ul>
+<button type="button" id="btn-project-6" class="btn-p">See Project</button>
+</section>`;
+
+document.getElementById('project-1').style.backgroundImage = projectsArray[0].image;
+document.getElementById('project-2').style.backgroundImage = projectsArray[1].image;
+document.getElementById('project-3').style.backgroundImage = projectsArray[2].image;
+document.getElementById('project-4').style.backgroundImage = projectsArray[3].image;
+document.getElementById('project-5').style.backgroundImage = projectsArray[4].image;
+document.getElementById('project-6').style.backgroundImage = projectsArray[5].image;
+
+document.getElementById('btn-project-1').addEventListener('click', () => {
+  popup(0);
+});
+document.getElementById('btn-project-2').addEventListener('click', () => {
+  popup(1);
+});
+document.getElementById('btn-project-3').addEventListener('click', () => {
+  popup(2);
+});
+document.getElementById('btn-project-4').addEventListener('click', () => {
+  popup(3);
+});
+document.getElementById('btn-project-5').addEventListener('click', () => {
+  popup(4);
+});
+document.getElementById('btn-project-6').addEventListener('click', () => {
+  popup(5);
+});
+document.getElementById('btn-project-work').addEventListener('click', () => {
+  popup(6);
+});
```

Comment:
- [ ] Please see my previous comment about hard-coded value.  Please keep your code dry by avoiding hard-coding index values. this causes a lot of repetition, unclean code, and is not maintanable.
In a case like this this would be much:
```suggestion
projectsArray.forEach( (_project, i) => {
  document.getElementById(`btn-project-${i}`).addEventListener('click', () => {
    popup(i);
  });
})
```

- another option would be to give the projects a unique class. that way you can use `querySelectorAll('bnt-project')`